drivers/edac/shell.c

```diff
@@ -140,7 +164,13 @@ static int cmd_inject_mask(const struct shell *shell, size_t argc, char **argv)
 	}
 
 	if (argc == 1) {
-		uint64_t mask = edac_inject_get_param2(dev);
+		uint64_t mask;
+
+		err = edac_inject_get_param2(dev, &mask);
+		if (err != 0) {
+			shell_error(shell, "Error getting mask");
```

Comment:
Maybe print the exact error as well with %d?

Reply:
I actually noticed that it's missing from many existing error cases, so a general improvement you could do as a separate patch